client/components/post-job/KeywordSelect.tsx

```diff
@@ -0,0 +1,18 @@
+import {skills} from '../../constants/skills';
```

Comment:
Do we have module path aliases setup? If so, you might want to use those so you don't have to refactor this import in the future.

Reply:
Would you mind opening an issue and proposing that? I think it's a change worth implementing, so we can use an issue to explain what it would accomplish/look like to those unfamiliar.